network/network.go

```diff
@@ -90,12 +93,37 @@ func (n *Network) CheckHealth() map[string]core.Health {
 			Intermediates: core.NewCertPool(n.trustStore.IntermediateCAs),
 		})
 		if err != nil {
-			results["tls"] = core.Health{
+			results[healthTLS] = core.Health{
 				Status:  core.HealthStatusDown,
 				Details: err.Error(),
 			}
 		} else {
-			results["tls"] = core.Health{
+			results[healthTLS] = core.Health{
+				Status: core.HealthStatusUp,
+			}
+		}
+	}
+	// auth_config checks that the node is correctly configured to be authenticated by others
+	nodeDID, err := n.nodeDIDResolver.Resolve()
+	if err != nil {
+		// can only happen when not in strictmode and autoNodeDIDResolver fails
+		results[healthAuthConfig] = core.Health{
+			Status:  core.HealthStatusUnknown,
+			Details: err.Error(),
+		}
```

Comment:
return results here and remove the else wrapper afterwards. Same can be done for the `else if` in the lines below.